src/Visitor.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace HexletPsrLinter;
+
+use PhpParser\Node;
+use PhpParser\NodeVisitorAbstract;
+
+class Visitor extends NodeVisitorAbstract
+{
+    private $errorFunctions = [];
+
+    public function leaveNode(Node $node)
+    {
+        $nodeClass = get_class($node);
+        if ($nodeClass == "PhpParser\\Node\\Stmt\\ClassMethod" && checkFunctionName($node->name) != true) {
```

Comment:
Поправил

On Fri, Aug 5, 2016 at 3:11 PM, Roman Lakhtadyr <notifications@github.com>
wrote:

> In src/Visitor.php
> <https://github.com/maximtop/hexlet-psr-linter/pull/1#discussion_r73682413>
> :
>
> > @@ -0,0 +1,28 @@
> > +<?php
> > +
> > +namespace HexletPsrLinter;
> > +
> > +use PhpParser\Node;
> > +use PhpParser\NodeVisitorAbstract;
> > +
> > +class Visitor extends NodeVisitorAbstract
> > +{
> > +    private $errorFunctions = [];
> > +
> > +    public function leaveNode(Node $node)
> > +    {
> > +        $nodeClass = get_class($node);
> > +        if ($nodeClass == "PhpParser\\Node\\Stmt\\ClassMethod" && checkFunctionName($node->name) != true) {
>
> а вообще можно же написать:
>
> if ($node instanceof \PhpParser\Node\Stmt\ClassMethod)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/maximtop/hexlet-psr-linter/pull/1/files/de77fd2c026aec74dad587f6eb61c5e151b5f7c4#r73682413>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQePwTdZ72W_eYc2MnuzXDD3MOipiqnIks5qcyhhgaJpZM4JdWB2>
> .
>



-- 
Sincerely yours,
Maxim Nikolaevich
mob. 8-926-934-08-25
email: maximtop@gmail.com